### PR TITLE
GRA-11 Restaurant Details

### PR DESCRIPTION
**Pull request by nedim-00**
- Implemented screen for restaurant details

- Small Navbar changes

### CODE REVIEW

**Review by KahrimanRr — APPROVED** [review]

File: src/components/Navbar/Nav.tsx [hunk 1]
```diff
@@ -19,15 +28,15 @@ const Nav = () => {
             }
           />
         </div>
-        <div className="flex items-center md:order-1">
+        {/* <div className="flex items-center md:order-1">
```
Comment (Tarik1700) [inline src/components/Navbar/Nav.tsx:31]: Remove commented code if you don't need it anymore

File: src/interfaces/IMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default interface IMenu {
```
Comment (Tarik1700) [inline src/interfaces/IMenu.tsx:1]: Interfaces designed only to be used for that specific file for props can and should stay in that file, any other interface that could be re-used in the future should go inside this interfaces folder.

File: src/interfaces/IRestaurantDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import IMenu from './IMenu';
```
Comment (Tarik1700) [inline src/interfaces/IRestaurantDetails.tsx:1]: same as above

File: src/pages/AfterLogin/Render.tsx [hunk 4]
```diff
@@ -1,55 +1,44 @@
...
-import SearchInput from "./SearchInput";
-import Cards from "./Cards";
-import Slider from "./Slider";
+import React from 'react';
+import SearchInput from './SearchInput';
+import Cards from './Cards';
+import Slider from './Slider';
 
+const Render = () => {
+  const slides = [
```
Comment (Tarik1700) [inline src/pages/AfterLogin/Render.tsx:9]: Maybe we should move these constants outside of these files, just to keep them clutter free, you can create another file somewhere called constants or data or something along those lines and import it from there.

File: src/pages/RestaurantInformation/RestaurantInformation.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+import pizza from '../../assets/images/restaurant/restaurant-menu/pizza.png';
+import hamburger from '../../assets/images/restaurant/restaurant-menu/hamburgers.png';
+import pancake from '../../assets/images/restaurant/restaurant-menu/pancakes.png';
+import IRestaurantDetails from '../../interfaces/IRestaurantDetails';
+import Menu from './Menu';
+import IMenu from '../../interfaces/IMenu';
+import Slider from './Slider';
+
+const RestaurantInformation = () => {
+  const restaurant: IRestaurantDetails = {
```
Comment (Tarik1700) [inline src/pages/RestaurantInformation/RestaurantInformation.tsx:13]: same as above, we should move these somewhere, maybe they're gonna be re-used in this prototype so its better if they are outside this file and exported

**Review by Tarik1700 — APPROVED** [review]

File: src/pages/RestaurantInformation/RestaurantInformation.tsx (hunk 5 above)
Comment (nedim-00) [reply src/pages/RestaurantInformation/RestaurantInformation.tsx:13]: Solved

**Pull request merged**